### PR TITLE
Improve front page doc

### LINKED ISSUE #3391: Improve front page <> documentation integration
Make it easier for a user when they go to worksheets.codalab.org to figure out how to get started / use documentation

### PR DESCRIPTION
**Pull request by yuqijin**
### Reasons for making this change

Make it easier for a user when they go to worksheets.codalab.org to figure out how to get started / use documentation

### Related issues

fixes #3391 

### Screenshots
Screen Shot 2021-04-29 at 6 05 50 PM

Screen Shot 2021-04-29 at 6 03 52 PM

Screen Shot 2021-05-07 at 5 09 19 AM

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: frontend/src/routes/HomePage.js [hunk 1]
```diff
@@ -96,6 +111,12 @@ class HomePage extends React.Component<{
                                     {auth.isAuthenticated &&
                                         this.renderButton('My Profile', '/users')}
                                 </div>
+                                <div>
+                                    {this.renderStartButton(
+                                        'Get Started',
```
Comment (percyliang) [inline frontend/src/routes/HomePage.js]: Could we make this appear on the same line as the other buttons?

Comment (yuqijin) [reply frontend/src/routes/HomePage.js]: Sure! Make the buttons on the same line now:
Screen Shot 2021-04-29 at 7 08 03 PM

Comment (epicfaace) [reply frontend/src/routes/HomePage.js]: Maybe we rename "Get started" to "Quickstart" or "Read Quickstart"? This is so it's clearer that that button goes to the documentation (technically, logging in / signing up is also a way to "get started")

Comment (yuqijin) [reply frontend/src/routes/HomePage.js]: thanks for the advice. The button has been renamed to "QUICKSTART"

File: frontend/src/routes/HomePage.js [hunk 2]
```diff
@@ -212,35 +233,20 @@ class HomePage extends React.Component<{
...
-                                >
-                                    Go To Repo
-                                </Button>
+                                <Typography variant='h5'>More Information</Typography>
                             </Grid>
                             <Grid item xs={12} sm={8} className={classes.textBox}>
                                 <Typography>
-                                    For more <b>advanced users</b>, the CodaLab documentation has a
-                                    wealth of information about many aspects of using the platform:
+                                    Acquire more knowledge you need to easily use the platform with
```
Comment (epicfaace) [inline frontend/src/routes/HomePage.js]:
```suggestion
                                    Learn more about how to use the platform with
```

**Review by epicfaace — APPROVED** [review]

File: docs/examples/quickstart.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
 # Tutorial: Quickstart
+ 
+[Go to Example Repo](https://github.com/codalab/worksheets-examples)
```
Comment (epicfaace) [inline docs/examples/quickstart.md]:
```suggestion
See the [examples GitHub repository](https://github.com/codalab/worksheets-examples/tree/master/00-quickstart) for the code related to this tutorial.
```

**Review by epicfaace — APPROVED** [review]

**Pull request merged**